### ISSUE TITLE
Interaction Wheel is missing mods it depends on? (Version 1.10.2)

### ISSUE THREAD

**Issue by katubug** [McJty/interactionwheel#9]
I'm getting an error that Interaction Wheel is missing a dependency mod, compatlayer 0.1.6 - but this isn't linked via Curse?

**Comment by McJty** [comment]
No, I have to readd the dependency. You can find compatlayer on curse though

**Comment by katubug** [comment]
Yep, I got it via one of your other mods. Thank you for all of the amazing
work that you do, by the way. My friends and I love your mods. You rock!
